scss/navbar.scss

```diff
@@ -0,0 +1,25 @@
+$nav-height: 90px;
+
+.navbar {
+  align-items: center;
+  background-color: $white;
+  display: flex;
+  height: $nav-height;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  z-index: 4;
```

Comment:
Why 4? Do you have elements with z-index of 2 or 3?

Reply:
I have elements with z-index 2, i will put 3 to make it sequential. There shouldn't be any change in it's behaviour